thunderbird-todoist/scripts/common.js

```diff
@@ -10,6 +10,10 @@ function loadMaillink() {
   return browser.storage.local.get('maillink').then(res => res.maillink === '1');
 }
 
+function loadFwdContent() {
```

Comment:
I think "fwdContent" is not a good name for this. It should be a bit more descriptive what it is doing, so maybe "IncludeMessageBody"?

Reply:
yes